tests/README.rst

```diff
@@ -80,6 +80,159 @@ To do that, run `podman pull quay.io/rhinstaller/anaconda-ci:master` or build
 it locally again.
 
 
+GH action integration
+---------------------
+
+This section explains our GH actions solution for testing pull requests to Anaconda.
+Explanation of parts are below.
+
+
+Explanation of the GH configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+master:
```

Comment:
Let's say "Pull requests for master", as we likely want to document non-PR workflows as well? (automatic container refreshes, COPR builds, etc.)